ivadomed/utils.py

```diff
@@ -674,7 +674,9 @@ def segment_volume(folder_model, fname_image, fname_roi=None):
         ds = imed_loaded_utils.filter_roi(ds, nb_nonzero_thr=context["slice_filter_roi"])
 
     if not context['unet_3D']:
-        print("\nLoaded {} {} slices..".format(len(ds), context['slice_axis']))
+        print(f"\nLoaded {len(ds)} {context['slice_axis']} slices..")
+    else:
+        print(f"\nLoaded {len(ds)} {context['slice_axis']} volumes of shape {context['length_3D']}")
```

Comment:
I would like to learn: why? :-)

Reply:
Hehe no reason, I think it's just what was on master? I merged quickly the other line is fine as well!